Source/Engine/PackageBuilder/SyntaxParser.cs

```diff
@@ -852,7 +900,12 @@ private Syntax ParsePatternReference(string patternName)
                         throw SyntaxError(TextResource.AttributesAreNotAllowedForStandardPattern, patternName);
                 }
                 else
-                    result = pattern.Body;
+                    result = pattern.Body switch
+                    {
+                        TokenSyntax token => new TokenSyntax(token.TokenKind, token.Text, token.IsCaseSensitive, token.TextIsPrefix, token.TokenAttributes),
+                        VariationSyntax variation => new VariationSyntax(variation.Elements, false),
```

Comment:
Make sure that VariationSyntax branch is covered by tests.

Reply:
Please use named parameter for bool literal:
```suggestion
                        VariationSyntax variation => new VariationSyntax(variation.Elements, checkCanReduce: false),
```